### PR TITLE
export passes in pdf

### PR DESCRIPTION
**Pull request by Tschonti**
exmaple pdf: 
[schbody_pass_export_1725295039684.pdf](https://github.com/user-attachments/files/16839827/schbody_pass_export_1725295039684.pdf)

the background image is stored in a envvar for now, should probably be stored with the periods.

### CODE REVIEW

File: apps/frontend/src/.env.example
Comment (FearsomeRover) [inline apps/frontend/src/.env.example:1]: This should be in the frontend folder, not in src, no?

**Review by FearsomeRover — APPROVED** [review]

File: apps/frontend/src/.env.example
Comment (Tschonti) [reply apps/frontend/src/.env.example:1]: Yeah probably, but it was here for some reason xd

**Comment by Tschonti** [comment]
Ne mergeld még, kicsit szépítem ma

**Comment by Tschonti** [comment]
New pass design just dropeed: 
[schbody_pass_export_1725374512064.pdf](https://github.com/user-attachments/files/16850561/schbody_pass_export_1725374512064.pdf)

**Comment by mozsarmate** [comment]
nice work!
could you make the labels a bit bigger?  anyways... i guess, i merge it, although we might have to change the a layout a bit, after our meeting tomorrow

**Pull request merged**